lib/ood_core/job/adapters/slurm.rb

```diff
@@ -178,6 +179,18 @@ def get_jobs(id: "", owner: nil, attrs: nil)
             return [{ id: id, state: 'undetermined' }]
           end
 
+          def accounts
+            user = Etc.getlogin
+            args = ['-np', 'show', 'accounts', 'withassoc', 'format=account,user', '-P']
+
+            [].tap do |accts|
+              call('sacctmgr', *args).each_line do |line|
+                acct, username = line.split('|')
+                accts << acct if username.to_s.chomp == user
+              end
+            end.uniq.map(&:upcase)
```

Comment:
Is it possible to not do upcase but only do upcase for OSC? I ask because Slurm down cases everything in the database, both users and accounts.

Reply:
Yea I was also wondering how to do this. some sort of environment variable `OOD_UPCASE_SLURM_ACCOUNTS` or similar?

Reply:
Ya an environment variable or cluster YAML config option.

Reply:
Went for an environment variable as that's an easier interface (this library doesn't read any config files yet safe for cluster files).